### DISCUSSION TITLE
HardLinks tag

### DISCUSSION THREAD

**Discussion by Minituff** [StuffAnThings/qbit_manage#633] [Ideas]
Hello,

I know Qbitmanage can tell me if I have no hardlinks using the `noHL` tag, but I would also love the ability to see if there is a hardlink-- maybe something like a `HL` tag?.

This way, I know the script has run and has indeed looked at my torrent and has validated its hardlinks. 
Currently, if I look at a torrent and I don't see a tag for `other` or any of my trackers, then I don't know if I have hardlinks or not.

Just an idea, thanks for considering.

**Comment by bakerboy448** [comment]
The script already provides various notifications including start and ended. Why do you need to look at qbit to determine if it ran?

why do you need to look at qbit often?

**Reply by Minituff** [reply]
Not a necessity or anything. I figured it might be helpful.

My use case is just to check out the Qbittorrent UI everyone once in a while instead of using notifications or checking the logs. That's the main appeal I have for Qbitmanage, is it allows me to be even more hands-off with my torrents.

It would honestly just give me peace of mind to open Qbit and see that my hardlinks are working.

Maybe I'm thinking about it so much right now since I just spent a few hours re-working my setup to get hardlinks working :)
